shared/version/api.go

```diff
@@ -393,6 +393,7 @@ var APIExtensions = []string{
 	"resources_disk_mounted",
 	"server_version_lts",
 	"oidc_groups_claim",
+	"auto_stateful_migration",
```

Comment:
`<thing>_<change>` format please, so `migration_stateful_default`

Reply:
Please follow the API extension commit structure ive mentioned to you before. Thanks